docs/soft-delete-implementation.md

```diff
@@ -0,0 +1,76 @@
+# PayloadCMS Soft Delete Implementation
+
+## Overview
+
+This document describes the implementation of PayloadCMS native soft delete functionality across target collections using the `trash: true` configuration option introduced in version 3.49.0.
+
+## Collections with Soft Delete Enabled
+
+The following collections now support soft delete functionality:
+
+- **Clinics** - Critical business data with relationships to doctors and treatments
+- **Doctors** - Referenced by reviews and treatment associations
+- **Treatments** - Core medical data with clinic and doctor relationships
+- **Medical Specialties** - Foundational data referenced by doctors and treatments
+- **Reviews** - Important audit trail and patient feedback data
+- **Media** - Files that may be referenced by multiple entities
+- **Posts** - Blog content that may be referenced for SEO
+- **Pages** - Static pages that may be referenced in navigation
+- **Tags** - Classification data referenced by multiple entities
+
+## Permission Matrix Update
+
+### Delete & Restore Access Control
+
+| Collection | Delete Permission | Restore Permission | Rationale |
+|------------|------------------|--------------------|-----------|
+| Clinics | Platform Staff Only | Platform Staff Only | Business critical data |
+| Doctors | Platform Staff Only | Platform Staff Only | Referenced by reviews/treatments |
+| Treatments | Platform Staff Only | Platform Staff Only | Core medical data |
+| Medical Specialties | Platform Staff Only | Platform Staff Only | Foundational data |
+| Reviews | Platform Staff Only | Platform Staff Only | Audit trail integrity |
+| Media | Platform Staff Only | Platform Staff Only | Referenced by multiple entities |
+| Posts | Platform Staff Only | Platform Staff Only | Content management |
+| Pages | Platform Staff Only | Platform Staff Only | Site structure |
+| Tags | Platform Staff Only | Platform Staff Only | Classification system |
+
+**Note**: PayloadCMS uses the existing `delete` access control for restore operations. No separate restore permission configuration is required.
```

Comment:
This note you can keep. Seems valuable but reference a source on payloaddocs to prove it. If you do not have prove. Remove it.